src/pages/index.js

```diff
@@ -1,129 +1,235 @@
 import UserInfo from "../components/UserInfo.js";
-import profileImagePath from "../images/jacques-cousteau.jpg";
 import logoPath from "../images/logo.svg";
+import Api from "../components/Api.js";
+import profileImagePath from "../images/jacques-cousteau.jpg";
 
-document.querySelector(".header__logo").src = logoPath;
 document.querySelector(".profile__image").src = profileImagePath;
+document.querySelector(".header__logo").src = logoPath;
 
-document.querySelector(".header__logo").addEventListener("load", () => {});
-document.querySelector(".profile__image").addEventListener("load", () => {});
-
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createCard(cardData);
-      cardSection.addItem(cardElement);
-    },
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "49b1b68f-7c8f-45c7-9c0e-4dcf6a213fcc",
+    "Content-Type": "application/json",
   },
-  ".cards"
-);
+});
 
 const userInfo = new UserInfo({
   nameSelector: ".profile__name",
   aboutSelector: ".profile__about",
   avatarSelector: ".profile__image",
 });
 
-const addpopup = document.getElementById("add-popup");
-const addForm = document.forms["add-form"];
-const titleInput = addForm.querySelector("#place");
-const urlInput = addForm.querySelector("#link");
-
-const editpopup = document.getElementById("edit-popup");
-const nameInput = document.querySelector("#name");
-const aboutInput = document.querySelector("#about");
-
-const profileName = document.querySelector(".profile__name");
-const profileAbout = document.querySelector(".profile__about");
-
-const openAddpopupButton = document.querySelector(".profile__add-button");
-const openEditpopupButton = document.querySelector(".profile__edit-button");
-
 const imagePopup = new PopupWithImage("#image-popup");
 const editProfilePopup = new PopupWithForm(
   "#edit-popup",
   handleProfileFormSubmit
 );
 const addCardPopup = new PopupWithForm("#add-popup", handleAddCardFormSubmit);
+const avatarPopup = new PopupWithForm("#avatar-popup", handleAvatarFormSubmit);
+const confirmPopup = new PopupWithConfirm("#confirm-popup");
 
 imagePopup.setEventListeners();
 editProfilePopup.setEventListeners();
 addCardPopup.setEventListeners();
+avatarPopup.setEventListeners();
+confirmPopup.setEventListeners();
 
-function handleProfileFormSubmit(formData) {
-userInfo.setUserInfo({
-  name: formData.name,
-  about: formData.about,
-});
-  editProfilePopup.close();
-}
+const cardSection = new Section(
+  {
+    items: [],
+    renderer: (item) => {
+      const cardElement = createCard(item);
+      cardSection.addItem(cardElement);
+    },
+  },
+  ".cards"
+);
 
-function handleAddCardFormSubmit(data) {
-  const cardElement = createCard(data);
-  cardSection.addItem(cardElement);
-  addCardPopup.close();
-}
+cardSection.setItems = function (items) {
+  this._items = items;
+};
```

Comment:
Please, delete the code from line 58 till 60 because it's not needed. You have the correct `setItems` method **inside** `Section`